freqtrade/exchange/binance.py

```diff
@@ -100,6 +100,7 @@ def stoploss(self, pair: str, amount: float, stop_price: float,
                 side=side,
                 amount=amount,
                 price=rate,
+                leverage=leverage,
```

Comment:
This is wrong - this `create_order` call is the ccxt one - ccxt does not have a "leverage" parameter on `createOrder`.

Reply:
Right, I just got it confused with the freqtrade `createOrder`